ql/src/test/results/clientnegative/authorization_update_noupdatepriv.q.out

```diff
@@ -6,4 +6,4 @@ POSTHOOK: query: create table auth_noupd(i int, j int) clustered by (j) into 2 b
 POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@auth_noupd
-FAILED: HiveAccessControlException Permission denied: Principal [name=user1, type=USER] does not have following privileges for operation QUERY [[SELECT] on Object [type=TABLE_OR_VIEW, name=default.auth_noupd], [UPDATE] on Object [type=TABLE_OR_VIEW, name=default.auth_noupd]]
```

Comment:
earlier it was checking for `UPDATE` + `SELECT` ; now it checks for `INSERT`+`SELECT`+`DELETE` 

this change might be unexpected to people working on authorization....not sure what we could do about it;
but I think in its current form; depending on the feature toggle we will be checking: UPDATE/SELECT+DELETE - which could be unexpected

Reply:
The split update can be turned off if this turn out to be an issue.